Twittimation/Commands/Command.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Twittimation.Commands
+{
+    public abstract class Command
+    {
+        public abstract string Name { get; }
+        public abstract List<string> RequiredArgs { get; }
+        public abstract List<string> OptionalArgs { get; }
+        public abstract Optional<string> OptionalRepeatedArg { get; }
+        public abstract string HelpInfo { get; }
+        public abstract string ExtendedHelp { get; }
+
+        public abstract void Go(string[] args);
+
+        public string CreateSyntaxString()
+        {
+            return $"Syntax: {Name}{string.Concat(RequiredArgs.Select(a => " <" + a + ">"))}{string.Concat(OptionalArgs.Select(a => " [<" + a + ">]"))}{(OptionalRepeatedArg.HasValue ? " [<" + OptionalRepeatedArg.Value + ">]...": "")}";
```

Comment:
This should be correctly indented. Commands shouldn't be to the right of the syntax or Help Text.